C2GUILauncher/src/Chivalry2Launchers.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace C2GUILauncher
+{
+    static class Chivalry2Launchers
+    {
+        public const string BinDir = "TBL\\Binaries\\Win64";
+        public const string GameBinPath = BinDir + "\\Chivalry2-Win64-Shipping.exe";
+        public const string OriginalLauncherPath = "Chivalry2Launcher-ORIGINAL.exe";
+
+        public static ProcessLauncher VanillaLauncher { get; } = new ProcessLauncher(OriginalLauncherPath, "");
+        public static ProcessLauncher ModdedLauncher { get; } = new ProcessLauncher(GameBinPath, BinDir, new string[] { "C2PluginLoader.dll" });
```

Comment:
Here I imagine you would insert all the core mod loading dlls. Or we can move the dll list initialization somewhere else of course